### DISCUSSION TITLE
Organization plan listed as "GitHub Free" instead of "GitHub Team" after upgrade.

### DISCUSSION THREAD

**Discussion by kanazuchijin** [community/community#65369] [GitHub Education]
### Hi everyone,

I recently upgraded an organization for my class from the Free version to Team. I received approval and confirmation that the request was processed. Yet, when I navigate to the settings of the organization --> Billing and Plans --> Current Plan the Free version is listed, not Team. When I navigate to https://education.github.com/discount_requests/application I see the organization has its status listed as "Approved." Any suggestions or thoughts? My students are unable to access the content they need for the course through their learning management system.

I have been using GitHub for years and never experienced an issue like this. I even have another organization for a different class for this semester that has its approval and the current plan, "Team," listed.

**Comment by arafajs** [comment]
This can happen if there is a delay in processing the upgrade. You can wait a few hours and then check again to see if the plan has been updated.
If the plan is still not updated after a few hours, then you can contact GitHub support to investigate the issue.
